models/customer/src/generator.ts

```diff
@@ -32,11 +32,15 @@ const generator = Generator<TCustomer>({
     dateOfBirth: null,
     companyName: fake((f) => f.company.name()),
     vatId: fake((f) => f.string.alphanumeric(12)),
-    addresses: fake(() => Address.random()),
+    addresses: fake(() => [Address.random()]),
```

Comment:
I don't know exactly how flexible we have to be, but I think addresses can eventually be just an empty array

Reply:
should be okay I think, then one could do: `Customer.random().addresses([]).buildGraphql()`

Reply:
sounds good!